docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -339,23 +339,14 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
-   "language": "python",
-   "name": "python3"
+   "display_name": "Python 3.9.12 64-bit",
+   "name": "python3912jvsc74a57bd05490e004a2fbce82e5eed04d33b6e654682831a3b87c276e4ea465354771c79b"
   },
   "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
    "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.8.12"
+   "version": ""
```

Comment:
I'm not exactly sure if these changes have any significant impact. I'm willing to just let them be merged but we can also arrange for the tutorials to be executed once on a UNIX system to try and minimize the differences here.